community/cloud-foundation/templates/cloudbuild/examples/cloudbuild_trigger.yaml

```diff
@@ -0,0 +1,47 @@
+# Example of the Cloud Build trigger template usage.
+#
+# In this example, a Build trigger for automated builds is created.
+#
+# Replace the following placeholders with valid values:
+#   <FIXME:buildtrigger_description> : trigger description
+#   <FIXME:project_id>  : PROJECT_ID of the project where the repo resides
+#   <FIXME:repo_name>   : a valid Cloud Source repository name
+#   <FIXME: branch_name>: a valid branch name in the source repository
+#   <FIXME:builder_name>: a cloud-builder supported on Google Cloud;
+#                         see details at 
+#                         https://cloud.google.com/cloud-build/docs/cloud-builders
+#   <FIXME:build_args>  : a list of build arguments supported by the builder;
+#                         enter each argument on a separate line as show below
+#   Example git clone build step:
+#     - name: 'gcr.io/cloud-builders/git'
+#       args:
+#         - clone
+#         - https://github.com/GoogleCloudPlatform/deploymentmanager-samples
+
+imports:
+  - path: templates/cloudbuild/trigger.py
+    name: trigger.py
+
+resources:
+  - name: mybuildtrigger
+    type: trigger.py
+    properties:
+      description: <FIXME:buildtrigger_description>
+      triggerTemplate:
+        repoSource:
```

Comment:
I'm getting an error when running this example:
```shell
"{\"ResourceType\":\"gcp-types/cloudbuild-v1:cloudbuild.projects.triggers.create\"\
    ,\"ResourceErrorCode\":\"400\",\"ResourceErrorMessage\":{\"code\":400,\"message\"\
    :\"Invalid JSON payload received. Unknown name \\\"repo_source\\\" at 'trigger.trigger_template':\
    \ Cannot find field.\",\"status\":\"INVALID_ARGUMENT\",\"details\":[{\"@type\"\
    :\"type.googleapis.com/google.rpc.BadRequest\",\"fieldViolations\":[{\"field\"\
    :\"trigger.trigger_template\",\"description\":\"Invalid JSON payload received.\
    \ Unknown name \\\"repo_source\\\" at 'trigger.trigger_template': Cannot find\
    \ field.\"}]}],\"statusMessage\":\"Bad Request\",\"requestPath\":\"https://cloudbuild.googleapis.com/v1/projects/ocsi-courseralab1-guestbook/triggers\"\
    ,\"httpMethod\":\"POST\"}}"
```

Reply:
Sorry bout this. Not sure how this escaped two reviewers eyes
Fix in commit `b806511`

Reply:
It's ok :) important thing is we caught it.